src/celengine/catentry.cpp

```diff
@@ -80,12 +80,16 @@ bool CatEntry::isInCategory(const std::string &s) const
 
 bool CatEntry::loadCategories(Hash *hash)
 {
+    const char *d = nullptr;
+    std::string td;
+    if (hash->getString("TrDomain", td) && !td.empty())
```

Comment:
TranslationFile ?

Reply:
Rather not, as `gettext` doesn't provide ability to choose particular translation file - at most we can provide catalog. So do you mean `TranslationCatalog`, avoiding domain as proxy?

Reply:
I just mean that we need a more meaningful name than TrDomain. TranslationDomain or TranslationCatalog are both ok.